front/cypress/e2e/proposal_posting_anonymous.cy.ts

```diff
@@ -53,40 +48,30 @@ describe('Initiatives with anonymous participation allowed', () => {
     cy.get('.e2e-topics-picker').find('button').eq(3).click();
 
     // set to anonymous
-    cy.get('#e2e-post-anonymously-checkbox').should('exist');
     cy.get('#e2e-post-anonymously-checkbox').click();
-    cy.wait(1000);
+    cy.get('#e2e-continue-anonymous-participation-btn').click();
 
     // save the form
-    cy.get('#e2e-initiative-publish-button').click({
-      force: true,
-    });
-    cy.wait(1000);
-
-    // confirm anonymous participation
-    cy.get('#e2e-continue-anonymous-participation-btn').should('exist');
```

Comment:
This line isn't needed. If we are able to click the button on the next line, we already verify that the button exists.